.github/workflows/ci.yml

```diff
@@ -24,15 +24,13 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           override: false
-      - name: Run clippy
-        with:
-          command: clippy
-          args: --verbose --release --workspace --examples --tests --all-features -- -D warnings
       - name: Run tests
+        env: 
+          RUSTFLAGS: "-D warnings"
```

Comment:
This should be enabled at the global `env` so that we have a more uniform calling convention.

Reply:
Ohh I did not see the other usages! My bad